tests/org.eclipse.swt.tests/JUnit Tests/org/eclipse/swt/tests/junit/Test_org_eclipse_swt_widgets_Button.java

```diff
@@ -425,6 +428,48 @@ public void test_setTextLjava_lang_String() {
 	button.setText("");
 }
 
+@Test
+public void test_traverseCheckButton() {
```

Comment:
This test fails on Linux.
https://download.eclipse.org/eclipse/downloads/drops4/I20220406-0600/testresults/html/org.eclipse.swt.tests_ep424I-unit-cen64-gtk3-java11_linux.gtk.x86_64_11.html

```
Test_org_eclipse_swt_widgets_Button | test_traverseCheckButton | Failure | N/A
java.lang.AssertionErrorat org.junit.Assert.fail(Assert.java:87)
at org.junit.Assert.assertTrue(Assert.java:42)
at org.junit.Assert.assertTrue(Assert.java:53)
at org.eclipse.swt.tests.junit.Test_org_eclipse_swt_widgets_Button.test_traverseCheckButton(Test_org_eclipse_swt_widgets_Button.java:448)
```

Reply:
@niraj-modi Did you check the build results before merging?
I was not able to check them because build results were removed already, probably as result of the merge. I wonder if the test consistently fails or if it is flacky.